scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
I already have a docker installed on ppc64le jenkins slave, so we can login into the machine and push image or we can create jenkins job.